io-modules/jsonio.py

```diff
@@ -34,7 +35,58 @@ def create_file(self, data: dict) -> bool:
         """
         with open(self.json_path, "w") as json_file:
             json.dump(data, json_file, indent=2, ensure_ascii=False)
-        
+
+        return True
+
+    def create_file_with_iterator(
```

Comment:
i dont know how to validate this, might need some test cases on this to make sure that it works as intended